packages/docs/docs/guides/rules-of-jsx.md

```diff
@@ -82,7 +82,7 @@ function App() {
 /**
  * Ensure the model's response is JSON.
  */
-function ValidateJsonOutput({ children }, { render }): string {
+function JsonOutput({ children }, { render }): string {
```

Comment:
the previous name seems more intuitive to me, what's the intent here?

Reply:
Just making it match the other instances where it was used. I could have gone either way. I'll make them all `ValidateJsonOutput` then.